source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -1519,28 +1501,26 @@ TEST_F(NiDAQmxDriverApiTests, AIVoltageChannel_CfgSampClkTimingAndAcquireData_Su
 TEST_F(NiDAQmxDriverApiTests, AIVoltageChannel_CfgSampClkTimingAndAcquireDataWithMinimumTimeout_SampleNotAvailableStatusIsReturned)
```

Comment:
I think sample clock timing is a precondition, not the function you are testing. Also, "finite" is more important than "sample clock".

This paraphrases the error name.

You can omit "is".

```suggestion
TEST_F(NiDAQmxDriverApiTests, AIFiniteAcquisition_ReadWithTimeoutTooSmall_SamplesNotYetAvailableStatusReturned)
```